### PR TITLE
Updated release notes for Open GEE 5.2.0

### PR DESCRIPTION
**Pull request by tst-nfarah**
Updated release notes for 5.2.0 by adding resolved issues (after 5.2.0 beta) and known issues.

### CODE REVIEW

**Comment by tst-nfarah** [comment]
I signed it

File: docs/geedocs/answer/7160000.html [hunk 1]
```diff
@@ -148,10 +149,95 @@
...
+        </tr>
+        <tr>
+          <td>380</td>
+          <td>Provider field in resource-view is blank</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>401</td>
+          <td>GEE commands are not in the path for sudo. </td>
+          <td>Add /opt/google/bin to the path for all users, including the super user</td>
```
Comment (tst-lsavoie) [inline docs/geedocs/answer/7160000.html]: Minor: another workaround is to specify the full path when running commands.  Not sure if it's worth adding that that.

File: docs/geedocs/answer/7160000.html [hunk 2]
```diff
@@ -148,10 +149,95 @@
...
+        </tr>
+        <tr>
+          <td>405</td>
+          <td>Vector layer preview not cleared in some situations</td>
+          <td>Reset the preview window to the correct state by either clicking on it or previewing another vector layer</td>
+        </tr>
+        <tr>
+          <td>407</td>
+          <td>Corrupt data warning when starting fusion</td>
+          <td>No current work around</td>
```
Comment (tst-lsavoie) [inline docs/geedocs/answer/7160000.html]: May want to state here that Fusion still operates correctly (as far as we can tell) so you can ignore the warning.

File: docs/geedocs/answer/7160000.html [hunk 3]
```diff
@@ -148,10 +149,95 @@
...
+        </tr>
+        <tr>
+          <td>423</td>
+          <td>Improve JPEG 2000 format support performance</td>
```
Comment (tst-lsavoie) [inline docs/geedocs/answer/7160000.html]: Minor: I'd reword this to something like "JPEG 2000 performance is slow" or something like that.

File: docs/geedocs/answer/7160000.html [hunk 4]
```diff
@@ -148,10 +149,95 @@
...
+        </tr>
+        <tr>
+          <td>453</td>
+          <td>Improve `check_server_processes_running` detection for uninstall</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>474</td>
+          <td>Running gee_check on some supported platforms reports that the platform is not supported</td>
+          <td>No current work around</td>
```
Comment (tst-lsavoie) [inline docs/geedocs/answer/7160000.html]: Maybe should say that they can ignore the failed test if they are using a supported platform and list the supported platforms (Ubuntu 14.04, Ubuntu 16.04, RHEL 7, and CentOS 7).

File: docs/geedocs/answer/7160000.html [hunk 5]
```diff
@@ -148,10 +149,95 @@
...
+        </tr>
+        <tr>
           <td>342</td>
           <td>Fusion crashes when opening an unsupported file type</td>
           <td>Re-open fusion and avoid opening unsupported file types.</td>
         </tr>
+        <tr>
+          <td>375</td>
+          <td>unbundle psycopg2</td>
+          <td>On Ubuntu 16.04 switch to python-psycopg2' instead of 'python2.7-psycopg2'.</td>
```
Comment (tst-lsavoie) [inline docs/geedocs/answer/7160000.html]: Missing tick mark before python-psycopg2.

Also, I think it would be good to make the title more descriptive: maybe "Invalid version of psycopg2 on Ubuntu 16.04".

File: docs/geedocs/answer/7160000.html [hunk 6]
```diff
@@ -210,6 +296,41 @@
           <td>Portable Globe Cutter fails with a Python error</td>
           <td>Fixed import errors in the cutter scripts</td>
         </tr>
+        <tr>
+          <td>381</td>
+          <td>Error running POI Search</td>
+          <td>Needed to declare the coding in headers of python script:# -- coding: utf-8 -</td>
+        </tr>
+        <tr>
+          <td>382</td>
```
Comment (tst-lsavoie) [inline docs/geedocs/answer/7160000.html]: I think these three documentation fixes (382, 413, and 414) can be removed.  We have a statement at the top (in the paragraph about open sourcing) that says a number of documentation changes were made.  I think that covers this, though I'd be open to other's thoughts.

File: docs/geedocs/answer/7160000.html [hunk 7]
```diff
@@ -81,6 +81,7 @@
         </tr>
       </tbody>
     </table>
+  <p>If you're upgrading from GEE 5.1.3, you may have to reinstall the prerequisite libraries after uninstalling 5.1.3 but before installing 5.2.0.</p>
```
Comment (tst-lsavoie) [inline docs/geedocs/answer/7160000.html]: This was not in the original ticket, but perhaps we should also point out that we made a number of testing updates.  That could go in the section near the top on Open Sourcing.

File: docs/geedocs/answer/7160000.html [hunk 8]
```diff
@@ -148,10 +149,95 @@
...
         </tr>
+        <tr>
+          <td>375</td>
+          <td>unbundle psycopg2</td>
+          <td>On Ubuntu 16.04 switch to python-psycopg2' instead of 'python2.7-psycopg2'.</td>
+        </tr>
+        <tr>
+          <td>380</td>
+          <td>Provider field in resource-view is blank</td>
+          <td>No current work around</td>
```
Comment (tst-lsavoie) [inline docs/geedocs/answer/7160000.html]: You can work around this by opening the resource to view the provider.  It might make it clearer to change the title to indicate that the provider field is blank for resources in the asset manager.  If you open the individual resource, the provider is shown.

File: docs/geedocs/answer/7160000.html [hunk 9]
```diff
@@ -148,10 +149,95 @@
...
+          <td>Fusion installer does not upgrade the asset root on RHEL 7</td>
+          <td>Upgrade the asset root manually by running the command that is printed when you try to start the fusion service</td>
+        </tr>
+        <tr>
+          <td>453</td>
+          <td>Improve `check_server_processes_running` detection for uninstall</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>474</td>
```
Comment (tst-lsavoie) [inline docs/geedocs/answer/7160000.html]: We should also add issue #456 in the known bugs.

File: docs/geedocs/answer/7160000.html (hunk 1 above)
Comment (tst-nfarah) [reply docs/geedocs/answer/7160000.html]: Sure. I'll add it.

File: docs/geedocs/answer/7160000.html (hunk 2 above)
Comment (tst-nfarah) [reply docs/geedocs/answer/7160000.html]: ok. How about 'No current work around but Fusion loads and runs correctly' ?

File: docs/geedocs/answer/7160000.html (hunk 3 above)
Comment (tst-nfarah) [reply docs/geedocs/answer/7160000.html]: Even if the original PR (423) title is 'Improve JPEG 2000 format support performance' ?

File: docs/geedocs/answer/7160000.html (hunk 4 above)
Comment (tst-nfarah) [reply docs/geedocs/answer/7160000.html]: ok. How about 'You can ignore the failed test if using a supported platform (Ubuntu 14.04, Ubuntu 16.04, RHEL 7 and CentOS 7)' ?

File: docs/geedocs/answer/7160000.html (hunk 5 above)
Comment (tst-nfarah) [reply docs/geedocs/answer/7160000.html]: ok for the '.' 
about the title, same question as before. We want to edit the title here but keep it as it is in the PR ? (375). The  PR title seems to point to a solution and not necessarily the issue.

File: docs/geedocs/answer/7160000.html (hunk 7 above)
Comment (tst-nfarah) [reply docs/geedocs/answer/7160000.html]: To clarify, you are proposing to move the 'If you're upgrading' to a new section near 'Open Sourcing' section and in that section add 'we made a number of testing updates' ? We should probably explain 'number of testing updates'.

File: docs/geedocs/answer/7160000.html (hunk 8 above)
Comment (tst-nfarah) [reply docs/geedocs/answer/7160000.html]: Ok

File: docs/geedocs/answer/7160000.html (hunk 9 above)
Comment (tst-nfarah) [reply docs/geedocs/answer/7160000.html]: Ok. I'll add it as well to the ticket.

File: docs/geedocs/answer/7160000.html (hunk 2 above)
Comment (tst-lsavoie) [reply docs/geedocs/answer/7160000.html]: Sounds good.

File: docs/geedocs/answer/7160000.html (hunk 3 above)
Comment (tst-lsavoie) [reply docs/geedocs/answer/7160000.html]: I haven't necessarily been keeping the issue title and the wording in the release notes the same, though that's not a bad idea.  The point I want to get across is that jp2 performance will be worse in 5.2.0 than it was in 5.1.3.

File: docs/geedocs/answer/7160000.html (hunk 4 above)
Comment (tst-lsavoie) [reply docs/geedocs/answer/7160000.html]: Looks good.

File: docs/geedocs/answer/7160000.html (hunk 7 above)
Comment (tst-lsavoie) [reply docs/geedocs/answer/7160000.html]: I think the "If you're upgrading..." sentence can stay here.  I only put the comment here because github doesn't let you comment on code that doesn't change.

On second thought I'm going to suggest not mentioning the testing since that's something end users don't really care about.  Which means you don't need to make any changes.

File: docs/geedocs/answer/7160000.html (hunk 5 above)
Comment (tst-lsavoie) [reply docs/geedocs/answer/7160000.html]: It currently says `On Ubuntu 16.04 switch to python-psycopg2'`.  It should say `On Ubuntu 16.04 switch to 'python-psycopg2'`.  The only change is to add an apostrophe.

The reason I'd suggest changing the title is that "unbundle psycopg2" sounds like an enhancement, but there is a known issue where some things don't work on Ubuntu 16.04 with the bundled psycopg2.  Maybe we should change the title here and on the issue.

File: docs/geedocs/answer/7160000.html (hunk 3 above)
Comment (tst-nfarah) [reply docs/geedocs/answer/7160000.html]: yeah, the title does not indeed reflect that. may be we leave it as is and if one interested to learn more they would look at the PR and understand the issue ? or we can change the title here ?

File: docs/geedocs/answer/7160000.html (hunk 4 above)
Comment (tst-nfarah) [reply docs/geedocs/answer/7160000.html]: ok

File: docs/geedocs/answer/7160000.html (hunk 3 above)
Comment (tst-lsavoie) [reply docs/geedocs/answer/7160000.html]: I'd like to change the title here - maybe "Slower JPEG2000 performance than 5.1.3" and then if they want to know more they can look at the issue on github.

File: docs/geedocs/answer/7160000.html (hunk 5 above)
Comment (tst-nfarah) [reply docs/geedocs/answer/7160000.html]: Added the missing '.
Changed the title in the release notes and added a comment/question in the 375 PR to eventually change its title.

File: docs/geedocs/answer/7160000.html (hunk 3 above)
Comment (tst-nfarah) [reply docs/geedocs/answer/7160000.html]: ok

File: docs/geedocs/answer/7160000.html (hunk 6 above)
Comment (tst-nfarah) [reply docs/geedocs/answer/7160000.html]: Yeah if these doc cleanup 'bugs' are not worth mentioning, i can remove them.

Comment (tst-lsavoie) [reply docs/geedocs/answer/7160000.html]: I vote to remove them.

Comment (tst-nfarah) [reply docs/geedocs/answer/7160000.html]: Done.

Comment (tst-lsavoie) [reply docs/geedocs/answer/7160000.html]: I could also see leaving just 382 as a sort of catch all, but you might want to update the issue number to "382 and others" or something like that.  If you try to list them all it will be a long list I think.
